src/components/Auth/ChangePassword.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { validatePassword } from '../Form/Validators'
+import client from '@/database/client'
+import ValidatedInput from '../Form/ValidatedInput'
+
+export default function ChangePassword() {
+    const [password, setPassword] = useState<string>('')
+    const [formValid, setValid] = useState<boolean>(false)
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        if (validatePassword(password)) {
+            setValid(true)
+        } else {
+            setValid(false)
+        }
+    }, [password])
+
+    const updatePassword = async () => {
+        const { error } = await client.auth.updateUser({
+            password: password,
+        })
+        if (!error) {
+            navigate('/')
+        } else {
+            alert("We're so sorry, an error occured.")
```

Comment:
We can look at flowbyte modal components for some non-native styles popups later

Reply:
Agreed, created #90 to track